markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -489,20 +492,24 @@ library PerpsAccount {
             PerpsMarketConfiguration.Data storage marketConfig = PerpsMarketConfiguration.load(
                 marketId
             );
-
-            uint256 numberOfWindows = marketConfig.numberOfLiquidationWindows(
+            maxNumberOfWindows = marketConfig.numberOfLiquidationWindows(
```

Comment:
By doing that you are assuming the latest market in the loop is the one that will require more liquidation windows (depends on market config). 
Previously, it picked the longest to process as # of windows